### PR TITLE
Eclipse Vert.x example

### PR DESCRIPTION
**Pull request by jponge**

### CODE REVIEW

File: examples/vertx/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Containerize a [Eclipse Vert.x](https://vertx.io/) application with Jib
+
+This is an example of how to easily build a Docker image for a [Eclipse Vert.x application](https://vertx.io/) with Jib.
+
+```shell
+/gradlew jibDockerBuild
```
Comment (briandealwis) [inline examples/vertx/README.md]: Missing leading `.` — `./gradlew`

File: examples/vertx/src/main/java/example/vertx/MainVerticle.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    @Override
+    public void start(Future<Void> startFuture) {
+        Router router = Router.router(vertx);
+
+        router.get("/").handler(this::hello);
+        router.get("/time").handler(this::now);
+
+        vertx.createHttpServer()
+                .requestHandler(router)
+                .listen(8080, ar -> {
```
Comment (briandealwis) [inline examples/vertx/src/main/java/example/vertx/MainVerticle.java]: Could you use a more meaningful variable name than `ar`?  I'm not that familiar with vert.x and can't really guess what this means. `allocatedRouter`?

File: examples/vertx/src/main/java/example/vertx/MainVerticle.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        logger.info("Time request from {}", context.request().remoteAddress());
+        JsonObject data = new JsonObject()
+                .put("powered-by", "vertx")
+                .put("current-time", System.currentTimeMillis());
+        context.response()
+                .putHeader("Content-Type", "application/json")
+                .end(data.encode());
+    }
+
+    public static void main(String[] args) {
```
Comment (briandealwis) [inline examples/vertx/src/main/java/example/vertx/MainVerticle.java]: Our coding style prefers that public static methods to appear at the top.  Is this the standard appearance of verticals?

File: examples/vertx/README.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+docker run -d --rm -p 8080:8080 vertx-jib-example
+```
+
```
Comment (briandealwis) [inline examples/vertx/README.md]: Could you include a _More Information_ section with links to both Jib (for people who come directly into this page from a search) and to Vert.x?

File: examples/vertx/README.md (hunk 1 above)
Comment (jponge) [reply examples/vertx/README.md]: woops

File: examples/vertx/src/main/java/example/vertx/MainVerticle.java (hunk 2 above)
Comment (jponge) [reply examples/vertx/src/main/java/example/vertx/MainVerticle.java]: `AsyncResult<T>`, we often use `ar`, but I can for sure have something more meaningful.

File: examples/vertx/src/main/java/example/vertx/MainVerticle.java (hunk 3 above)
Comment (jponge) [reply examples/vertx/src/main/java/example/vertx/MainVerticle.java]: Ok, I can move it.

File: examples/vertx/README.md (hunk 4 above)
Comment (jponge) [reply examples/vertx/README.md]: Sure!

**Comment by jponge** [comment]
@briandealwis here you go!

File: examples/vertx/README.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+```shell
+./gradlew jibDockerBuild
+
+docker run -d --rm -p 8080:8080 vertx-jib-example
+```
+
+## More information
+
+Learn [more about Jib](https://github.com/GoogleContainerTools/jib).
+Learn [more about Eclipse Vert.x](https://vertx.io).
```
Comment (TadCordle) [inline examples/vertx/README.md]: nit: Need another newline or a double space after the first line so they don't end up on the same line.

**Comment by jponge** [comment]
Done :-)

**Pull request merged**